### ISSUE TITLE
Windows Defender Issue

### ISSUE THREAD

**Issue by sinanislekdemir** [pal1000/mesa-dist-win#38]
Hi, 

On Windows 10, Windows Defender complains about finding 3 trojans inside the latest release package;

Can you check and clean if possible?

**Comment by pal1000** [comment]
Most likely FP. You can try recently 20.1.1 release.

**Comment by sinanislekdemir** [comment]
I hope this helps, I created a fresh Win10 machine in a Virtualbox (and obviously Virtualbox is irrelevant to this problem).

Here is the problem:

https://www.youtube.com/watch?v=AOEX_27FnIc

(The video is not public, just visible with the link)

**Comment by pal1000** [comment]
I looked at that video and it appears both detections are exclusive to tests stack. If you want, you can remove all `.exe` files from archive. Programs that use Mesa need only `.dll` and  `.cmd` files.

**Comment by sinanislekdemir** [comment]
I tested with F Secure and AVG which were clean. So likely some Microsoft related glitch. 
Sorry, I mainly use Linux and MacOS and just had to test my library within a Virtualbox and thanks to your efforts here I was still able to test OpenGL 3.x in a VirtualBox Windows and, I confirmed that my library works as expected under Windows as well.

Closing the issue and taking a note for myself for future reference and thank you for keeping things maintained with this. :pray:
